### DISCUSSION TITLE
Can one route be forwarded to another route in Apisix？

### DISCUSSION THREAD

**Discussion by yijieshusheng-git** [apache/apisix#11157] [Q&A]
Can one route be forwarded to another route in Apisix？
在apisix中一个路由可以转发给另一个路由吗

**Comment by ThiagoFigaro** [comment]
Some options:
1 Use the parameter URI override (plugin proxy-rewrite) and insert the path of the new route
{
  "uri": "/route1",
  "name": "1",
  "methods": [
    "GET"
  ],
  "plugins": {
    "proxy-rewrite": {
      "uri": "/route2"
    }
  }

2 Use the redirect plugin to config some rule using request parameter - https://apisix.apache.org/docs/apisix/plugins/redirect/

3 Workflow plugin to set rules even after the upstream response - https://apisix.apache.org/docs/apisix/plugins/workflow/
There are many
